packages/teams-js/test/private/legacy.spec.ts

```diff
@@ -78,6 +82,32 @@ describe('AppSDK-privateAPIs', () => {
     });
   });
 
+  describe('joinedTeams.isSupported function', () => {
+    const utils = new Utils();
+    it('joinedTeams.isSupported should return false if the runtime says joinedTeams is not supported', () => {
+      utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
```

Comment:
Based on the implementation of isSupported, I think it's useful to have a couple more tests that cover the other places in the code that lead to a false return value. i.e., `supports: { teams: {} }`, `supports: { teams: { fullTrust: {} } }`

Reply:
If we'd had isSupported tests before we moved the API into a sub-namespace, one of those would fail after the change, which is ideal.

Reply:
Sure, will add it in a gif